CodeStarterCamp_Week1/Step3.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  second.swift
+//  CodeStarterCamp_Week1
+//
+//  Created by Jeong Min Lee on 2/13/24.
+//
+
+import Foundation
+
+func drawPepero(bodyWidth width: Int,pockyBody body: String ,peperoTopping topping: String, barLength length : Int) {
+    print ("""
+            <정보>
+            길이: \(width)
+            몸통: \(body)
+            토핑: \(topping)
+            막대길이: \(length)
+            """)
```

Comment:
```suggestion
    print ("""
           <정보>
           길이: \(width)
           몸통: \(body)
           토핑: \(topping)
           막대길이: \(length)
           """)
```

여러줄 문자열을 사용하셨군요 👍

닫는 큰 따옴표를 기준으로 문자열이 들어가기 때문에 현재의 상태로도 문제없이 동작합니다.

하지만 개인적으로는 여는 `"""`와 닫는 `"""`를 정렬하는 것이 보기 편한 것 같습니다. 꼭 그렇게 하라는 것은 아니고, 한 번 해보시고 보기 편한 쪽을 사용하시면 될 것 같아요.